solutions/baseline/variables.tf

```diff
@@ -0,0 +1,297 @@
+  default     = "private"
+  validation {
+    error_message = "The specified `endpoint_type` can be `private` or `public` only."
+    condition     = contains(["private", "public"], var.cloud_monitoring_endpoint_type)
+  }
+}
+
+variable "cloud_monitoring_metrics_filter" {
+  type = list(object({
+    type = string
+    name = string
+  }))
+  description = "To filter on custom metrics, specify the IBM Cloud Monitoring metrics to include or exclude. [Learn more](https://cloud.ibm.com/docs/monitoring?topic=monitoring-change_kube_agent#change_kube_agent_inc_exc_metrics)"
+  default     = [] # [{ type = "exclude", name = "metricA.*" }, { type = "include", name = "metricB.*" }]
+  validation {
+    condition     = alltrue([for filter in var.cloud_monitoring_metrics_filter : can(regex("^(include|exclude)$", filter.type))])
+    error_message = "The specified `type` for the `cloud_monitoring_metrics_filter` is not valid. Specify either `include` or `exclude`. If the value for `type` is not specified, no metrics are included or excluded."
+  }
+}
+
+variable "cloud_monitoring_agent_tags" {
+  type        = list(string)
+  description = "A list of the tags to associate with the metrics that the IBM Cloud Monitoring agent collects. To add the cluster name as a tag, use the `cloud_monitoring_add_cluster_name` variable."
+  default     = []
+  nullable    = false
+}
+
+variable "cloud_monitoring_add_cluster_name" {
+  type        = bool
+  description = "Whether to attach a tag to log messages. Set to `true` to configure the IBM Cloud Monitoring agent to attach a tag that contains the cluster name to all log messages."
+  default     = true
+}
+
+variable "cloud_monitoring_agent_name" {
+  description = "The name of the IBM Cloud Monitoring agent that is used to name the Kubernetes and Helm resources on the cluster. If a prefix input variable is passed, the name of the IBM Cloud Monitoring agent is prefixed to the value in the `<prefix>-<name>` format."
+  type        = string
+  default     = "sysdig-agent"
+}
+
+variable "cloud_monitoring_agent_namespace" {
+  type        = string
+  description = "The namespace to deploy the IBM Cloud Monitoring agent in. Default value: `ibm-observe`."
+  default     = "ibm-observe"
+  nullable    = false
+}
+
+variable "cloud_monitoring_agent_tolerations" {
+  description = "The list of tolerations to apply to the IBM Cloud Monitoring agent. The default operator value `Exists` matches any taint on any node except the master node. [Learn more](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)"
+  type = list(object({
+    key               = optional(string)
+    operator          = optional(string)
+    value             = optional(string)
+    effect            = optional(string)
+    tolerationSeconds = optional(number)
+  }))
+  default = [{
+    operator = "Exists"
+    },
+    {
+      operator : "Exists"
+      effect : "NoSchedule"
+      key : "node-role.kubernetes.io/master"
+  }]
```

Comment:
Projects is not liking this :/ 
As a workaround can you add as one line?
![image](https://github.com/user-attachments/assets/2cdb2bc5-a25e-48f9-8f63-375f81bbdc27)

Reply:
Hang on, that default value seems odd?? I see it is in the root module too. Can we check this??